components/Card.js

```diff
@@ -32,9 +31,22 @@ export default function Card({ props }) {
       }
     >
       <div>
-        <div className="flex flex-row justify-between items-center">
-          {dataset.logo && (
-            <img src={dataset.logo} alt="next" className="mr-2" />
+        <div className="flex justify-between items-center">
+          {dataset.image ? (
+            <img
+              src={dataset.image}
+              alt="dataset"
+              width="100px"
+              height="120px"
```

Comment:
We will want to make the size smaller I guess but it's good for now, let's get that in!

Reply:
What's the recommended size? prob 100x100